backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/ElasticSearchConstants.java

```diff
@@ -22,4 +22,6 @@
 public interface ElasticSearchConstants {
     String WRITE_ALIAS = "injectWriteAlias";
     String READ_ALIAS = "injectReadAlias";
+
+    String MAILBOX_INDEX = "mailboxIndex";
```

Comment:
If you do go in that direction, we need a `MAILBOX_READ_ALIAS` and `MAILBOX_WRITE_ALIAS`

For extended possibilities, we never work directly with indexes but rather with aliases (abstraction allowing to dynamically change of indexes <3 )